S2VX.Game/Editor/ToolState/HoldNoteToolState.cs

```diff
@@ -30,7 +30,7 @@ public class HoldNoteToolState : S2VXToolState {
             Child = PreviewContainer;
         }
 
-        public override bool OnToolClick(ClickEvent _) {
+        public override bool OnToolClick(ClickEvent e) {
```

Comment:
Are we not using the _ naming convention for unused variables anymore? I assume it's to follow CA1725: Parameter names should match base declaration

Reply:
Yep and yep